Automattic/wp-calypso
# App Banner: Remove Unnecessary Logic for Activity Log

#### Changes proposed in this Pull Request

In #26286, @roccotripaldi added a bit of code to prevent the Activity Log display the App Banner. It was necessary then, but it's no longer necessary as the Activity Log has been moved out of stats in #26941. :)

#### Testing instructions

Visit the Activity Log and verify the app banner doesn't appear, but confirm it does in Stats. Information on how to trigger the banner: #16124

<img width="574" alt="Screenshot 2019-07-10 at 22 25 53" src="https://user-images.githubusercontent.com/43215253/61006217-b1b38c00-a361-11e9-8c29-90ceaa4bce54.png">

Noticed whilst creating #34582
